fabtests/common/shared.c

```diff
@@ -2915,7 +2915,11 @@ int ft_sync()
 
 	if (opts.dst_addr) {
 		if (!(opts.options & FT_OPT_OOB_SYNC)) {
-			ret = ft_tx(ep, remote_fi_addr, 1, &tx_ctx);
+			ret = ft_sendmsg(ep, remote_fi_addr, 1, &tx_ctx, FI_DELIVERY_COMPLETE);
```

Comment:
Do all providers support delivery completion w/o first asking for it?

Reply:
The info will use the op_flags of hints (via fi_alter_tx_attr), so we don't know whether provider supports DC without application requesting it. I see many tests in fabtests use FI_TRANSMIT_COMPLETE and FI_INJECT_COMPLETE flags in fi_sendmsg without specifying them in the hints, i.e. provider may ignore them if it doesn't support those completion semantics. Does DC have to be special in this case? I would expect that if a provider doesn't support DC it may just ignore this flag